src/vs/workbench/contrib/codeActions/browser/codeActionsContribution.ts

```diff
@@ -52,11 +52,6 @@ const codeActionsOnSaveSchema: IConfigurationPropertySchema = {
 		}
 	],
 	markdownDescription: nls.localize('editor.codeActionsOnSave', 'Run CodeActions for the editor on save. CodeActions must be specified and the editor must not be shutting down. Example: `"source.organizeImports": "explicit" `'),
-	type: 'object',
```

Comment:
Does this break intellisense for the setting in the json editor? 

Also maybe we can use json schema's either type support to say that the value can either be an object or an array

Reply:
does not, but added specific typings, and intellisense working fine here.

Reply:
Just want to confirm if this is as expected:

What's being suggested in Insiders for random code action
<img width="846" alt="image" src="https://github.com/microsoft/vscode/assets/876920/8d77e433-1240-4635-acd3-bcdbef67a56d">

What's being suggested in OSS w/ this change for known code action
<img width="693" alt="image" src="https://github.com/microsoft/vscode/assets/876920/ab9d2a86-08d5-4e14-abb8-8e1c6df5e5e1">

What's being suggested in OSS w/ this change for random code action
<img width="789" alt="image" src="https://github.com/microsoft/vscode/assets/876920/1e37f199-80b8-4118-a464-1bebba23481f">

It feels like a regression/behavior change.

Reply:
Fixed by adding it back in! Keep in mind that with quotes and without quotes will give the following responses anyways. 

<img width="877" alt="Screenshot 2023-10-05 at 1 23 06 PM" src="https://github.com/microsoft/vscode/assets/54879025/24cc951f-f80d-46f6-9855-47645726f01f">

<img width="883" alt="Screenshot 2023-10-05 at 1 23 23 PM" src="https://github.com/microsoft/vscode/assets/54879025/194d255d-e428-4f61-b902-60787098b4ca">